applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAX.java

```diff
@@ -220,6 +221,12 @@ private void initialize(final String pluginVersion, final String sdkKey, final R
             targetingYearOfBirthToSet = null;
         }
 
+        if ( locationCollectionEnabledToSet != null )
```

Comment:
nit, but for consistency and when I put the different plugins' code side-by-side, can we order the logic the same way we do on [Unity](https://github.com/AppLovin/Unity-Plugin/blob/master/Android/Android-Unity-Plugin/src/main/java/com/applovin/mediation/unity/MaxUnityPlugin.java#L136)? Unity is usually the benchmark for other plugins.

`locationCollectionEnabledToSet` is before yob and gender.

Reply:
I think iOS is fine though

Reply:
Right, I don't know why I put it in there.    Will reorder and push.